src/main/java/lotto/domain/Rank.java

```diff
@@ -18,12 +21,10 @@ public enum Rank {
     public static Rank returnRank(int matchCount){
         Rank[] ranks = values();
 
-        for (Rank rank : ranks){
-            if(rank.matchCount == matchCount)
-                return rank;
-        }
-
-        return Rank.MISS;
+        return Arrays.stream(ranks)
+                .filter(rank -> rank.matchCount == matchCount)
```

Comment:
```suggestion
                .filter(rank -> rank.match(matchCount))
```
enum도 클래스와 같이 메시지를 보내는 방식으로 구현하는 것은 어떨까?